src/ConnectivityWizard/ConnectivityWizard.ts

```diff
@@ -0,0 +1,29 @@
+import logger from '../Utils/Logger';
+import { KoaRouterFactory } from './Routes/KoaRouterFactory';
+import { KoaAppFactory } from './KoaAppFactory';
+import Koa from 'koa';
+import getPort from 'get-port';
+import { TestType } from './Entities/ConnectivityTest';
+import { URL } from 'url';
+
+export class ConnectivityWizard {
+  private readonly bind_port: number = 3000;
+  private readonly range_size: number = 500;
+  private app: Koa;
+
+  public async init(options: Map<TestType, URL>): Promise<void> {
+    const routesFactory = new KoaRouterFactory(options);
+    this.app = await new KoaAppFactory(routesFactory).createApp();
```

Comment:
It violates ISP and DIP. If I implement a new app factory (express based), I will have to change `ConnectivityWizard` too. Please inject deps via the constructor.